File: Admin_p/Admin_p.Web/Modules/Identity/AspNetUser/AspNetUserForm.cs
using Serenity.ComponentModel;

namespace Admin_p.Identity.Forms;

[FormScript("Identity.AspNetUser")]
[BasedOnRow(typeof(AspNetUserRow), CheckNames = true)]
public class AspNetUserForm
{
    public string Name { get; set; }
    public string NormalizedName { get; set; }
    public string ConcurrencyStamp { get; set; }
}

File: Admin_p/Admin_p.Web/Modules/ServerTypes/Identity/AspNetUserForm.ts
import { PrefixedContext, StringEditor, initFormType } from '@serenity-is/corelib';

export interface AspNetUserForm {
    Name: StringEditor;
    NormalizedName: StringEditor;
    ConcurrencyStamp: StringEditor;
}

export class AspNetUserForm extends PrefixedContext {
    static readonly formKey = 'Identity.AspNetUser';
    private static init: boolean;
    
    constructor(prefix: string) {
        super(prefix);
        if (!AspNetUserForm.init)  {
            AspNetUserForm.init = true;
            
            var w0 = StringEditor;

            initFormType(AspNetUserForm, [
            'Name', w0,
            'NormalizedName', w0,
            'ConcurrencyStamp', w0,
            ]);
        }
    }
}